app/models/manageiq/providers/nuage/network_manager/event_catcher/messaging_handler.rb

```diff
@@ -21,13 +22,20 @@ def on_container_start(container)
     raise MiqException::MiqHostError, "Timeout connecting to AMQP endpoint #{@url}"
   end
 
+  def on_container_stop(container)
+    container.close_sockets
+  end
+
   def on_connection_open(connection)
     # In case connection test was requested, close the connection immediately.
     connection.container.stop if @test_connection
   end
 
-  def on_connection_error(_connection)
-    raise MiqException::MiqInvalidCredentialsError, "Connection failed due to bad username or password"
+  def on_connection_error(connection)
+    c = connection.condition
+    msg = "#{self.class.log_prefix} AMQP on_connection_error: #{c}"
```

Comment:
Can you condense l35 and l36 into a single one, since `c` is not used anywhere else.

Reply:
Done, thanks